devices/common_patches/gd32e23x.yaml

```diff
@@ -0,0 +1,5 @@
+RCU:
+  VKEY:
+    _modify:
+      KEY:
+        access: write-only
```

Comment:
Nit: add a newline at the end of the file (and likewise for the other YAML files you've added).